e2e-tests/e2e-tests-utils/src/test/java/org/evomaster/e2etests/utils/GraphqlTestBase.java

```diff
@@ -0,0 +1,220 @@
+package org.evomaster.e2etests.utils;
+
+import org.apache.commons.io.FileUtils;
+import org.evomaster.client.java.controller.EmbeddedSutController;
+import org.evomaster.client.java.controller.InstrumentedSutStarter;
+import org.evomaster.client.java.controller.api.dto.SutInfoDto;
+import org.evomaster.client.java.controller.internal.SutController;
+import org.evomaster.client.java.instrumentation.ClassName;
+import org.evomaster.core.Main;
+import org.evomaster.core.output.OutputFormat;
+import org.evomaster.core.output.compiler.CompilerForTestGenerated;
+import org.evomaster.core.problem.graphql.GraphqlIndividual;
+import org.evomaster.core.remote.service.RemoteController;
+
+import org.evomaster.core.search.Individual;
+import org.evomaster.core.search.Solution;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.platform.launcher.listeners.TestExecutionSummary;
+
+import java.io.File;
+import java.time.Duration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public abstract class GraphqlTestBase extends TestBase{
```

Comment:
`GraphqlTestBase`, `TestBase` and `RestTestBase` must all be refactored, by avoiding copy&paste of duplicated code